### PR TITLE
{bio}[system/system] MixMHC2pred v1.2

### PR DESCRIPTION
**Pull request by scicomp-moffitt**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MixMHC2pred/MixMHC2pred-1.2.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/GfellerLab/MixMHC2pred/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['839ebf71db3eb399b22e7522b71dc0e071ef9419ee0ab60dc4627d65264d9698']
+
+postinstallcmds = [
+    "chmod +rx %(installdir)s/%(name)s_unix",
+    "mv %(installdir)s/%(name)s %(installdir)s/%(name)s_mac",
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/m/MixMHC2pred/MixMHC2pred-1.2.eb:20]:
```suggestion
    "mv %(installdir)s/%(name)s %(installdir)s/%(name)s_mac",
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
This should fix
```
MixMHC2pred-1.2.eb:20:62: W299 trailing whitespace
```

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/MixMHC2pred/MixMHC2pred-1.2.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+easyblock = 'PackedBinary'
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/m/MixMHC2pred/MixMHC2pred-1.2.eb:2]:
```suggestion
easyblock = 'PackedBinary'
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @scicomp-moffitt!

**Pull request merged**